### PR TITLE
Recruitment status filters

### PR DESCRIPTION
**Pull request by Elkrival**
This pr adds the recruitment status filters. The not recruited status can be different values and to account for that the control flow takes into account 4 possible scenarios. 
1. only recruited is selected
2. only non recruited is selected
3. both are selected
4. none are selected

As per the client, the recruited filter should be on by default, and the included and excluded filters have been removed. 

[functionality video](https://drive.google.com/file/d/1HVAyzRg0ozhlWtW_X8PK3cVS35ml_ARU/view?usp=sharing)

### CODE REVIEW

File: server/controllers/chartsDataController/chartsDataController.test.js [hunk 1]
```diff
@@ -94,6 +89,17 @@ describe('chartsDataController', () => {
                   LA: { label: 'LA', value: '1' },
                   MA: { label: 'MA', value: '1' },
                 },
+                recruitment_status: {
+                  'Not recruited': {
+                    label: 'Not Recruited',
+                    value: '0',
+                  },
+
```
Comment (mikestone14) [inline server/controllers/chartsDataController/chartsDataController.test.js]:
```suggestion
```

File: server/controllers/chartsDataController/chartsDataController.test.js [hunk 2]
```diff
@@ -146,10 +152,16 @@ describe('chartsDataController', () => {
...
-                included_excluded: {
-                  Included: { label: 'Included', value: '0' },
-                  Excluded: { label: 'Excluded', value: '0' },
-                  Missing: { label: 'Missing', value: '0' },
+                recruitment_status: {
+                  'Not recruited': {
+                    label: 'Not Recruited',
+                    value: '1',
+                  },
+
```
Comment (mikestone14) [inline server/controllers/chartsDataController/chartsDataController.test.js]:
```suggestion
```

**Review by mikestone14 — COMMENTED** [review]
In the video, why do the sites displayed in the chart change when the recruitment filters are changed? Shouldn't those only change when the sites related filters change?

**Comment by Elkrival** [comment]
> In the video, why do the sites displayed in the chart change when the recruitment filters are changed? Shouldn't those only change when the sites related filters change?

@mikestone14 

This is because the subject id's for the filters assessments do not match the subject id's from data that's already imported. How does this affect sites displayed on the chart? The way this affects sites displayed on the chart is due to not having data of any participants for a particular site that meet the request, no participant data, means no site data being displayed on the chart.

File: server/controllers/chartsDataController/chartsDataController.test.js (hunk 2 above)
Comment (Elkrival) [reply server/controllers/chartsDataController/chartsDataController.test.js]: [1fea662](https://github.com/AMP-SCZ/dpdash/pull/743/commits/1fea6621d3e3a4989876f80f19c46ebe33d1acac)

File: server/controllers/chartsDataController/chartsDataController.test.js (hunk 1 above)
Comment (Elkrival) [reply server/controllers/chartsDataController/chartsDataController.test.js]: [1fea662](https://github.com/AMP-SCZ/dpdash/pull/743/commits/1fea6621d3e3a4989876f80f19c46ebe33d1acac)

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**